msg/CMakeLists.txt

```diff
@@ -160,6 +160,7 @@ set(msg_files
 	vtol_vehicle_status.msg
 	wheel_encoders.msg
 	wind_estimate.msg
+	rpm.msg
```

Comment:
We try to keep the list sorted alphabetically.

Reply:
sry - fixed